addons/mail/static/tests/thread/thread_tests.js

```diff
@@ -427,6 +427,32 @@ QUnit.test(
     }
 );
 
+QUnit.test("display channel name in chat window for public channels to all users", async () => {
```

Comment:
Would name test "can join public channel from channel mention link".

This makes test more relevant at a macro level for Discuss, making this test more maintainable even if we have or want to rewrite Discuss code from scratch.